sofa-boot-starter/pom.xml

```diff
@@ -304,6 +303,7 @@
                     <destFile>${jacoco.path}</destFile>
                     <dataFile>${jacoco.path}</dataFile>
                     <sessionId>jacoco_coverage</sessionId>
+                    <append>true</append>
                     <excludes>
```

Comment:
@NeGnail Is this `excludes` necessarily?

Reply:
@ujjboy    it is unnecessary.  my next pr will delete it.